pyvista/utilities/reader.py

```diff
@@ -1478,29 +1530,83 @@ def set_active_time_point(self, time_point):  # noqa: D102
         self.set_active_time_value(self.time_values[time_point])
 
 
+class DICOMReader(BaseReader):
+    """DICOM Reader for reading ``.dcm`` files.
+
+    This reader reads a single file or a path containing a several ``.dcm``
+    files (DICOM stack).
+
+    Parameters
+    ----------
+    path : str
+        Path to the single DICOM (``.dcm``) file to be opened or the directory
+        containing a stack of DICOM files.
+
+    Raises
+    ------
+    FileNotFoundError
+        If ``path`` is not a DICOM file (``.dcm``) or a directory containing
+        DICOM files.
+
+    Examples
+    --------
+    >>> import pyvista
+    >>> from pyvista import examples
+    >>> path = examples.download_dicom_stack(load=False)
+    >>> reader = pyvista.DICOMReader(path)
+    >>> dataset = reader.read()
+    >>> dataset.plot(volume=True, zoom=3, show_scalar_bar=False)
+
+    """
+
+    _class_reader = _vtk.vtkDICOMImageReader
+
+    def __init__(self, path: str):
+        """Initialize reader from DICOM file (*.dcm) or folder of DICOM files."""
+        super().__init__(path)
+
+        if os.path.isfile(path):
+            self._set_filename(path)
+        elif os.path.isdir(path):
+            if not os.listdir(path):
+                raise FileNotFoundError(f"No DICOM (*.dcm) files found in directory\n\n{path}")
+
+            file_types = set(pathlib.Path(file_).suffix for file_ in os.listdir(path))
+            file_types.discard('')
+
+            self._set_directory(path)
+        else:
+            raise FileNotFoundError(
+                f"{self.__class__.__name__} only accepts a single DICOM file"
+                f" (*.dcm) or a directory containing DICOM files. Path '{path}' is invalid"
+                " or does not exist."
+            )
```

Comment:
You end up here if path doesn't exist. Error message should probably be something more generic about the file/folder not existing ("given path is neither a file nor folder")

Reply:
Fixed in 7af391cd504166733c5eb82b4133ab2c4b0b29ef